Methods with ViUInt8 parameters are broken in nidigital API

### Description of issue

Methods with ViUInt8 parameters are broken in nidigital API. It's because ViUInt8 is not defined in [_visatype.py](https://github.com/ni/nimi-python/blob/master/generated/nidigital/_visatype.py).

Code generator needs to be updated.

### System report

`python -c "import nidigital; nidigital.print_diagnostic_information()"` output

```
OS:
    Name:      Windows
    Version:   10.0.16299
    Bits:      64
Driver:
    Name:      NI-Digital Pattern Driver
    Version:   Unknown
Module:
    Name:      nidigital
    Version:   0.1.1.dev0
Python:
    Version:   3.6.8 (tags/v3.6.8:3c6b436a57, Dec 24 2018, 00:16:47) [MSC v.1916 64 bit (AMD64)]
    Bits:      64
    Is_Venv:   True
    Installed Packages:
        zipp==0.6.0
        wcwidth==0.1.7
        six==1.12.0
        scipy==1.3.1
        pytest==5.2.1
        pyparsing==2.4.2
        py==1.8.0
        pluggy==0.13.0
        packaging==19.2
        numpy==1.17.3
        nitclk==0.1.0.dev0
        niscope==1.1.3
        nifgen==1.1.3
        nidigital==0.1.1.dev0
        nidcpower==1.1.2
        more-itertools==7.2.0
        importlib-metadata==0.23
        future==0.18.1
        enum34==1.1.6
        colorama==0.4.1
        attrs==19.3.0
        atomicwrites==1.3.0
        setuptools==40.8.0
        pip==19.0.3
```

### Steps to reproduce issue

``` 
with nidigital.Session(resource_name='my_device', options='Simulate=1, DriverSetup=Model:6570') as session:
    session.write_static(1)
```

Run the above snippet and observe this exception:
```
        vi_ctype = _visatype.ViSession(self._vi)  # case S110
        channel_list_ctype = ctypes.create_string_buffer(self._repeated_capability.encode(self._encoding))  # case C010
>       state_ctype = _visatype.ViUInt8(state)  # case S150
E       AttributeError: module 'nidigital._visatype' has no attribute 'ViUInt8'
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -21,6 +21,7 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 * ### ALL
     * #### Added
+        * `ViUInt8` is now a valid type in APIs
     * #### Changed
     * #### Removed
 * ### NI-DMM

--- a/build/templates/_visatype.py
+++ b/build/templates/_visatype.py
@@ -9,6 +9,7 @@ These are aliased directly to ctypes types so can be used directly to call into 
 
 ViChar = ctypes.c_char
 ViInt8 = ctypes.c_int8
+ViUInt8 = ctypes.c_uint8
 ViInt16 = ctypes.c_int16
 ViUInt16 = ctypes.c_uint16
 ViInt32 = ctypes.c_int32

--- a/generated/nidcpower/_visatype.py
+++ b/generated/nidcpower/_visatype.py
@@ -9,6 +9,7 @@ These are aliased directly to ctypes types so can be used directly to call into 
 
 ViChar = ctypes.c_char
 ViInt8 = ctypes.c_int8
+ViUInt8 = ctypes.c_uint8
 ViInt16 = ctypes.c_int16
 ViUInt16 = ctypes.c_uint16
 ViInt32 = ctypes.c_int32

--- a/generated/nidigital/_visatype.py
+++ b/generated/nidigital/_visatype.py
@@ -9,6 +9,7 @@ These are aliased directly to ctypes types so can be used directly to call into 
 
 ViChar = ctypes.c_char
 ViInt8 = ctypes.c_int8
+ViUInt8 = ctypes.c_uint8
 ViInt16 = ctypes.c_int16
 ViUInt16 = ctypes.c_uint16
 ViInt32 = ctypes.c_int32

--- a/generated/nidmm/_visatype.py
+++ b/generated/nidmm/_visatype.py
@@ -9,6 +9,7 @@ These are aliased directly to ctypes types so can be used directly to call into 
 
 ViChar = ctypes.c_char
 ViInt8 = ctypes.c_int8
+ViUInt8 = ctypes.c_uint8
 ViInt16 = ctypes.c_int16
 ViUInt16 = ctypes.c_uint16
 ViInt32 = ctypes.c_int32

--- a/generated/nifake/_visatype.py
+++ b/generated/nifake/_visatype.py
@@ -9,6 +9,7 @@ These are aliased directly to ctypes types so can be used directly to call into 
 
 ViChar = ctypes.c_char
 ViInt8 = ctypes.c_int8
+ViUInt8 = ctypes.c_uint8
 ViInt16 = ctypes.c_int16
 ViUInt16 = ctypes.c_uint16
 ViInt32 = ctypes.c_int32

--- a/generated/nifgen/_visatype.py
+++ b/generated/nifgen/_visatype.py
@@ -9,6 +9,7 @@ These are aliased directly to ctypes types so can be used directly to call into 
 
 ViChar = ctypes.c_char
 ViInt8 = ctypes.c_int8
+ViUInt8 = ctypes.c_uint8
 ViInt16 = ctypes.c_int16
 ViUInt16 = ctypes.c_uint16
 ViInt32 = ctypes.c_int32

--- a/generated/nimodinst/_visatype.py
+++ b/generated/nimodinst/_visatype.py
@@ -9,6 +9,7 @@ These are aliased directly to ctypes types so can be used directly to call into 
 
 ViChar = ctypes.c_char
 ViInt8 = ctypes.c_int8
+ViUInt8 = ctypes.c_uint8
 ViInt16 = ctypes.c_int16
 ViUInt16 = ctypes.c_uint16
 ViInt32 = ctypes.c_int32

--- a/generated/niscope/_visatype.py
+++ b/generated/niscope/_visatype.py
@@ -9,6 +9,7 @@ These are aliased directly to ctypes types so can be used directly to call into 
 
 ViChar = ctypes.c_char
 ViInt8 = ctypes.c_int8
+ViUInt8 = ctypes.c_uint8
 ViInt16 = ctypes.c_int16
 ViUInt16 = ctypes.c_uint16
 ViInt32 = ctypes.c_int32

--- a/generated/nise/_visatype.py
+++ b/generated/nise/_visatype.py
@@ -9,6 +9,7 @@ These are aliased directly to ctypes types so can be used directly to call into 
 
 ViChar = ctypes.c_char
 ViInt8 = ctypes.c_int8
+ViUInt8 = ctypes.c_uint8
 ViInt16 = ctypes.c_int16
 ViUInt16 = ctypes.c_uint16
 ViInt32 = ctypes.c_int32

--- a/generated/niswitch/_visatype.py
+++ b/generated/niswitch/_visatype.py
@@ -9,6 +9,7 @@ These are aliased directly to ctypes types so can be used directly to call into 
 
 ViChar = ctypes.c_char
 ViInt8 = ctypes.c_int8
+ViUInt8 = ctypes.c_uint8
 ViInt16 = ctypes.c_int16
 ViUInt16 = ctypes.c_uint16
 ViInt32 = ctypes.c_int32

--- a/generated/nitclk/_visatype.py
+++ b/generated/nitclk/_visatype.py
@@ -9,6 +9,7 @@ These are aliased directly to ctypes types so can be used directly to call into 
 
 ViChar = ctypes.c_char
 ViInt8 = ctypes.c_int8
+ViUInt8 = ctypes.c_uint8
 ViInt16 = ctypes.c_int16
 ViUInt16 = ctypes.c_uint16
 ViInt32 = ctypes.c_int32

